tsconfig.json

```diff
@@ -6,19 +6,13 @@
     "sourceMap": true /* Generates corresponding '.map' file. */,
     "outDir": "./build" /* Redirect output structure to the directory. */,
     "declaration": true,
-    "declarationDir": "./build/types",
     "strict": true /* Enable all strict type-checking options. */,
     "esModuleInterop": true /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */,
     "skipLibCheck": true /* Skip type checking of declaration files. */,
     "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */,
     "moduleResolution": "node",
-    "baseUrl": "./src",
-    "paths": {
-      "@bravura": ["fonts/loadStatic"],
-      "@gonville": ["fonts/loadStatic"],
-      "@petaluma": ["fonts/loadStatic"],
-      "@custom": ["fonts/loadStatic"]
```

Comment:
These were the magic path aliases that we were using this summer to implement dynamic font loading. However, Rodrigo and I couldn't get it working during our integration phase. The multiple entry files solution we have now is more explicit (there's a little less magic).